modules/machineset-yaml-nutanix.adoc

```diff
@@ -78,36 +91,37 @@ endif::infra[]
       providerSpec:
         value:
           apiVersion: machine.openshift.io/v1
```

Comment:
There is another field introduced in 4.13: `bootType`, not sure if it still needs to be added in the doc. Although it supports Legacy, SecureBoot and UEFI, but you must use the Legacy boot type in OpenShift Container Platform 4.13, which is the default bootType, same effect as not set this field. It shows as below by default in the default machineset yaml.
`          bootType: ""`

Reply:
IIRC, bootType, in particular, was added for supporing windows worker machines. @yanhua121 can you take a look?

Reply:
Ok, I think the most recent commit has everything fixed up, and this PR is now only targeting 4.13+